integration-core/src/main/resources/integration-core-log-messages.properties

```diff
@@ -0,0 +1,42 @@
+core.user.get.userid.fail=Failed to retrieve user information. User ID: {0}
+core.user.get.username.fail=Failed to retrieve user information. Username: {0}
+core.user.get.useremail.fail=Failed to retrieve user information. Email: {0}
+core.user.key.store.password.not.found=Failed to open keystore file from {0}
+core.user.key.store.password.not.found.solution=Please make sure that file permissions are right \
+  the keystore password isn't wrong \
+  or keystore file format isn't wrong
+core.user.key.store.password.not.retrieved=Fail to retrieve the keystore password. Application: {0}
+core.bootstrapcontext.no.integration.found=No integrations found to bootstrap
+core.bootstrapcontext.polling.agent.health.check=Polling AGENT health check
+core.bootstrapcontext.fail.bootstrap.integration=Fail to bootstrap the integration {0}
+core.bootstrapcontext.polling.stopped=Polling stopped
+core.bootstrapcontext.integration.successfully=Integration {0} bootstrapped successfully
+core.bootstrapcontext.fail.bootstrap.integration.retrying=Fail to bootstrap the integration {0}, but retrying... 
+core.bootstrapcontext.verify.new.integrations=Verify new integrations
+core.bootstrapcontext.shutting.down.integration=Shutting down integration {0}
+core.integrationlogging.integration.health.status=Integration: {0}, health status: {1}
+core.integrationlogging.fail.log.integration.health=Failed to log the {0} Integration Health
+core.integrationlogging.application.health.status=Application Health Status: {0}
+core.integrationlogging.fail.log.application.health=Failed to log the Application Health
+core.integrationlogging.fail.perform.health.logging=Fail to perform the health logging
+core.integration.bridge.exception.handler.unable.post.stream=The Integration Bridge was unable to post to stream {0}\
+   due to error code {1}
+core.integration.bridge.exception.handler.invalid.message=Invalid message posted to stream {0}. Instance: {1}
+core.integration.bridge.exception.handler.fail.post.message = Fail to post message
+core.integration.bridge.exception.handler.fail.update.streams = Fail to update streams
+core.integration.bridge.exception.handler.fail.notify.owner= Fail to notify owner
+core.integration.bridge.exception.handler.update.instance.notify = User notified about the instance updated
+core.integration.bridge.impl.no.configured.streams=No streams configured to instance {0} and configuration {1}
+core.integration.bridge.impl.user.posted.message=User {0} posted message to stream {1}
+core.integration.bridge.impl.no.streams= There are no streams configured to instance
+core.integration.bridge.impl.no.streams.solution = There are no Bot in the room(stream) \
```

Comment:
Wrong solution. There are no streams configured for the instance, instance owner should re-configure this instance or remove it from the system.